android/src/main/java/com/oney/WebRTCModule/ReactBridgeUtil.java

```diff
@@ -0,0 +1,23 @@
+package com.oney.WebRTCModule;
+
+import com.facebook.react.bridge.ReadableMap;
+import com.facebook.react.bridge.ReadableType;
+
+public class ReactBridgeUtil {
+
+    public static String getMapStrValue(ReadableMap map, String key) {
```

Comment:
@paweldomas, since you're not modifying an existing file, I'd rather we have javadocs please. I personally added javadocs to methods in existing files which I modified substantially (e.g. JavaScript, Objective-C). What do you think?